### PR TITLE
Run prettier formatter on all markdown files

### PR DESCRIPTION
**Pull request by algomaster99**

### CODE REVIEW

**Comment by shcheklein** [comment]
@algomaster99 thanks!

File: static/docs/user-guide/external-dependencies.md [hunk 1]
```diff
@@ -127,16 +127,16 @@ The command above creates an <abbr>import stage</abbr> specified in DVC-file
 ### Expand to see resulting DVC-file
 
 ```yaml
-...
+---
```
Comment (jorgeorpinel) [inline static/docs/user-guide/external-dependencies.md]: This is a bug.

File: static/docs/user-guide/external-dependencies.md [hunk 2]
```diff
@@ -167,18 +167,18 @@ dependency is specified.
...
```
Comment (jorgeorpinel) [inline static/docs/user-guide/external-dependencies.md]: Same here.

**Comment by jorgeorpinel** [comment]
@algomaster99 I know this has been merged but see the comments above. That's another problem we currently have with `prettier`...

**Comment by jorgeorpinel** [comment]
@algomaster99 I know this has been merged but see the comments above. That's another problem we currently have with `prettier`... Can you please undo those changes in a new PR? I already have a PR open and its under review.

Then let's address the bug separately? Created #532 for you 😬

**Comment by jorgeorpinel** [comment]
Actually never mind, I just opened PR #533 from a new `hotfix` branch so let's just go straight to trying to fix the bug (#532) please.

**Pull request merged**